src/main.cpp

```diff
@@ -101,6 +102,8 @@ int main(int argc, char *argv[]){
             auto program = parse_to_cfg_transformer<std::set<std::string>>(prog);
             time_func("Cublas creation: ", 
                 create_cublas);
+            time_func("Variable reduction: ", 
+                reduce_variables<std::set<std::string>>, program.entryNodes);
```

Comment:
Only implemented on Cublas?

Reply:
The base branch doesnt have CuBool, but it would be trivial to add variable reduction to CuBool aswell.